src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -79,7 +79,8 @@
                     </button>
                     <div id="run-report-status" v-if="runningStatus" class="text-secondary mt-2">
                         {{ runningStatus }}
-                        <a @click.prevent="checkStatus" href="#">Check status</a>
+                        <a @click.prevent="checkStatus" href="#">Check status</a> /
```

Comment:
I think we can get rid of the `Check status` link now - it was only every a stopgap until the running reports stuff, and this ticket, was done.